server.go

```diff
@@ -80,7 +81,7 @@ func ServerPingInterval(interval time.Duration) ServerOption {
 // communicating with osquery over the socket at the provided path. If
 // resolving the address or connecting to the socket fails, this function will
 // error.
-func NewExtensionManagerServer(name string, sockPath string, opts ...ServerOption) (*ExtensionManagerServer, error) {
+func NewExtensionManagerServer(name string, version string, sockPath string, opts ...ServerOption) (*ExtensionManagerServer, error) {
```

Comment:
Instead of making `version` a required parameter (which would be a breaking API change), can you make a `ServerOption` that allows setting the version?